### PR TITLE
Track all actions that are relevant (reject, approve, etc)

### LINKED ISSUE #16: Track other actions, like reject and approve
Currently we're only tracking `create` actions, we should also track:

- reject
- approve
- fuzzy?
- others?

### PR DESCRIPTION
**Pull request by psrpinto**
Fixes #16 

We're now tracking the following actions

- `create`: Create a translation
- `approve`: Approve a translation
- `reject`: Reject a translation
- `mark_fuzzy`: Mark a translation as fuzzy
- `request_changes`: Changes were requested on a translation

### CODE REVIEW

**Review by trymebytes — CHANGES_REQUESTED** [review]
The post meta key [here](https://github.com/WordPress/wporg-gp-translation-events/blob/3f2842970e03a20fc3adbed07f828fc129f78999/includes/class-wporg-gp-translation-events-translation-listener.php#L66) and [here](https://github.com/WordPress/wporg-gp-translation-events/blob/3f2842970e03a20fc3adbed07f828fc129f78999/includes/class-wporg-gp-translation-events-translation-listener.php#L72) need to be updated to `_event_start` and `_event_end` respectively.

File: includes/class-wporg-gp-translation-events-translation-listener.php [hunk 1]
```diff
@@ -2,55 +2,80 @@
 
 class WPORG_GP_Translation_Events_Translation_Listener {
 	const ACTIONS_TABLE_NAME = 'wp_wporg_gp_translation_events_actions';
-	const ACTION_CREATED = 'created';
+	const ACTION_CREATE = 'create';
+	const ACTION_APPROVE = 'approve';
+	const ACTION_REJECT = 'reject';
+	const ACTION_MARK_FUZZY = 'mark_fuzzy';
```
Comment (trymebytes) [inline includes/class-wporg-gp-translation-events-translation-listener.php:9]: Are we still tracking fuzzy statuses? Also, we need to track `changesrequested` status.

Comment (psrpinto) [reply includes/class-wporg-gp-translation-events-translation-listener.php:9]: `changesrequested` is now being tracked, good catch.

For tracking fuzzy, I opened #29.

**Comment by psrpinto** [comment]
Thanks for reviewing @trymebytes! I think I've addressed all your remarks.

**Review by trymebytes — APPROVED** [review]
Looks good to me, thank you!

**Pull request merged**